lib/jsmidgen.js

```diff
@@ -408,6 +408,15 @@ var Midi = {};
 		return bytes;
 	};
 
+	File.prototype.setTicks = function(num) {
+		// Do nothing in case of invalid input
+		if (!num || typeof num !== 'number') {
+			return;
+		}
+
+		this.HDR_SPEED = String.fromCharCode((num/256),  num%256);
```

Comment:
Yes sure, that way the overall code (for the ticks) becomes more readable as well.